info/description.md

```diff
@@ -1,18 +1,18 @@
 Ingots in each consignment are numbered in the row from A1 to Z9 as
-A1, A2,..., A9, B1, B2, ..., Z9. Each consignment are marked by the last ingots in it.
-So you can define the quantity of ingots my marks.
-Each daily report written as consignments of marks in string separated by commas.
-So you can count the total quantity of ingots for a day.
+A1, A2,..., A9, B1, B2, ..., Z9. Each consignment is marked by the last ingots placed in it.
+So you can define the quantity of ingots by the marks made on the side.
+Each daily report written as consignments of these marks in string separated by commas.
+With this, you'll be able to find the total quantity of ingots produced in a day.
 
-The full report contain daily reports for several days.
+The full report contain daily reports covering several days.
 Each report is given with a date in the next format:
 `YYYY-MM-DD`, where YYYY is year, MM is month, DD is day.
-Date and report are separated by whitespace. Each date-report are placed on separated lines.
+Dates and reports are separated by whitespaces. Each date-report is placed on separate lines.
 
-You are given a full report as a multiline text and two dates.
-You should calculate the total quantity ingots for the days between given dates (**including**).
+You are given a full report as multiline text with two dates.
```

Comment:
I think "with two lines" and "and two lines" are different, because we have text and dates separately.